### ISSUE TITLE
[Bug] Broken function in BottledExp (CraftExpContainer)

### ISSUE THREAD

**Issue by arturek1666** [Zrips/BottledExp#23]
I had noticed that the plugin from version BottledExp3.0.0.3 the function:
  CraftExpContainer:
```
    # When this set to true there will be option to craft special expbottle by placing empti glass bottle into crafting
    # You can consume it by right clicking it
    Use: true
```
Dosn't actually work as instead of right clicking to get the collected exp in my case lvl 49 I got lvl 3. It's a bit broken. I was hoping someone else will report it but looks like I had to take it into my own hands. 
This is a video to show how it's broken:
https://www.youtube.com/watch?v=vBTkzjLyw_U
This video shows how it should work:
https://www.youtube.com/watch?v=BSv8YqnRzW0
For the best Results I also reset the config completely and just enabled that one option to find this bug as long as   CraftExpContainer is on true should be easy to recreate.

Server Engine:
Purpur
Server Version:
1.17.1 - This bug should work on any supported version anyway
Plugin Version 
3.0.1.1 
The issue is from version:
3.0.0.3 and up to 3.0.1.1

**Comment by Silarious** [comment]
I am also encountering the same bug.
Using older plugin version(3.0.0.2) works as arturek mentioned.

Server Engine:
Pufferfish Purpur 
Server Version:
1.17.1
Plugin Version:
3.0.1.1

Another bug I encountered is the when CraftExpContainer is false I can still craft normal experience bottles and it does not take away any experience causing a infinite experience glitch. This happens on both versions I tested 3.0.1.1 & 3.0.0.2
